qiskit/circuit/annotation.py

```diff
@@ -84,6 +84,106 @@
 .. autoclass:: QPYSerializer
 .. autoclass:: QPYFromOpenQASM3Serializer
 .. autoclass:: OpenQASM3Serializer
+
+
+Examples
+========
+
+A toy block-collection transpiler pass
```

Comment:
I'm wondering if there is a better word besides "toy" we can use here. I agree it's a toy example but for the official documentation I'm not sure if we want to refer to it as something else.